en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md

```diff
@@ -0,0 +1,214 @@
+    - The below example uses the `js-cookie` library for cookie parsing. You can use any cookie-parsing library of your choice. 
+
+We recommend you retrieve the user info from the cookie and subsequently clear the cookie. Below is a sample code snippet that you can include in your post-login path for this purpose:
+
+```
+    import Cookies from 'js-cookie';
+
+    // Read userinfo cookie value.
+    const encodedUserInfo = Cookies.get('userinfo')
+
+    // Decode the value. 
+    const userInfo = JSON.parse(atob(encodedUserInfo))
+
+    // Store the value in a preferred browser-based storage if needed.
+
+    // Clear the cookie.
+    Cookies.remove('userinfo', { path: <post-login-path> })
+```
+#### Obtain user info via a GET endpoint
+
+Choreo's Managed Authentication provides a GET endpoint, `/auth/userinfo`, in addition to the userinfo cookie that it sets after a successful login. You can utilize this endpoint to query logged-in user info, and it can also serve as a way to check the logged-in state of a user.'
+
+Below is an example of a request to this endpoint:
+
+```
+const response = await fetch('/auth/userinfo')
+```
+
+If the user has logged in, the server will send a 200 OK response with the user info in the body in JSON format. However, if the user has not logged in, the server will send a 401 Unauthorized response.
+
+### Logout
+
+To enable Choreo-managed logout capability to your web application, you can implement a login button redirecting users to `/auth/logout` along with the `session_hint` cookie value when clicked. You can use the following code snippet or any custom button component from your preferred UI component library.
+
+!!! note
+    - It is recommended to clear the user info (if stored) at logout.
+    - The below example uses the `js-cookie` library for cookie parsing. You can use any cookie-parsing library of your choice.   
+    
+```
+<button onClick={async () => {
+    window.location.href = `/auth/logout?session_hint=${Cookies.get('session_hint')}`;
+}}>Login</button>`
+```
+
+When a user clicks on the logout button, Choreo will clear the session cookies and redirect the users to the OIDC logout endpoint of the configured Identity Provider (if available).  
+
+### API calls
+
+You can invoke Choreo APIs within the same organization as your web application using the relative path /choreo-apis/<api-suffix>, regardless of whether managed authentication is enabled for the web application or not.
+
+For example, if the API URL is `https://2d9ec1f6-2f04-4127-974f-0a3b20e97af5-dev.e1-us-east-azure.choreoapis.dev/rbln/item-service/api-e04/1.0.0`, the `<api-suffix>` would be `/rbln/item-service/api-e04/1.0.0`. From your Single Page Application, you can invoke the API using `/choreo-apis/rbln/item-service/api-e04/1.0.0` relative path.
```

Comment:
Should mention this relative URL can be found in web app's dependencies > connection page

Reply:
Resolved with [c5726a1](https://github.com/wso2/docs-choreo-dev/pull/1183/commits/c5726a15bc558bf96b3fb05ab1f27c4838baf301)